bigchaindb/db/utils.py

```diff
@@ -123,5 +126,6 @@ def drop(assume_yes=False):
             logger.info('Done.')
         except r.ReqlOpFailedError:
             raise exceptions.DatabaseDoesNotExist('Database `{}` does not exist'.format(dbname))
+        
     else:
-        logger.info('Drop aborted')
+      logger.info('Drop aborted')
```

Comment:
Indentation needs to be four spaces. As per [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation). The [pep8](https://pypi.python.org/pypi/pep8) tool outputs the following error:

```bash
bigchaindb/db/utils.py:131:7: E111 indentation is not a multiple of four
```